app/views/user_signed_out.scala.html

```diff
@@ -32,15 +31,17 @@
     </p>
 }
 
+@paragraphBodyContent = {
```

Comment:
Why you're not using `paragraphBody` component?

Reply:
Because it only accepts `String` as input and I need to have link inside.